tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -68,6 +73,23 @@ steps:
       flub --help
       flub commands
 
+- ${{ if eq(parameters.usesPrettier, true) }}:
+  # package.json files are reformatted by upcoming update version steps, then will be reformatted by prettier. We check
+  # them here because otherwise legitimate formatting problems would be hidden by the forced reformatting in CI.
+  - task: Bash@3
+    displayName: Check package.json formatting
+    inputs:
+      targetType: 'inline'
+      workingDirectory: ${{ parameters.buildDirectory }}
+      script: |
+        if [ -f "lerna.json" ]; then
+          # Run on all the release group packages, if in a release group
+          npx lerna exec prettier -- --check package.json
```

Comment:
Is it safe for us to use `npx` for calls like this in pipelines? Don't we run the risk of downloading incompatible versions of lerna this way?

Reply:
Adding `--no-install` is a good idea. I'll do that.